lambda/index.js

```diff
@@ -42,9 +42,8 @@ const AddItemIntentHandler = {
     let speakOutput;
 
     // Get item name from request
-    const item = Alexa.getSlotValue(handlerInput.requestEnvelope, 'item');
-
-
+    let item = Alexa.getSlotValue(handlerInput.requestEnvelope, 'item');
+    item = item.charAt(0).toUpperCase() + item.slice(1); // Make first letter uppercase
```

Comment:
Is this purely for aesthetics or does it have any functional value?

Reply:
Purely aesthetics. It looks ugly having all lowercase if you look at the shopping list in Home Assistant. And because you normally add objects and no verbs to a shopping list this is a good improvement.

Reply:
Yes that's also true. I think so far it's better to only make the first letter uppercase. Of couse the word "Tomaten" would be lowercase then but still better than everything in lowercase. I think that would fit most of the part. To correctly case all words it would require some kind of library I think.